components/Comments/Comment.vue

```diff
@@ -160,6 +186,38 @@
       }
     },
     mounted () {
+      // add the light gray color to the triangle at the left of the comment
+      if (this.depth > 0) {
+        const el = this.$refs.commentMain
+        el.classList.remove('depth-zero')
+        el.classList.add('depth-one')
+
+        var addRule = (function (style) {
+          var sheet = document.head.appendChild(style).sheet;
+          return function (selector, css) {
+            var propText = typeof css === "string" ? css : Object.keys(css).map(function (p) {
+              return p + ":" + (p === "content" ? "'" + css[p] + "'" : css[p]);
+            }).join(";");
+            sheet.insertRule(selector + "{" + propText + "}", sheet.cssRules.length);
+          };
+        })(document.createElement("style"));
```

Comment:
Maybe add the source as comment here.

Reply:
I found the post where I got this function from. I added it as comment.

Reply:
This is against the principles of vue (do not work in the Dom is one) and seems to be over complicated. 

But why do you need to change the pseudo styles in the first place? 

There should be a better way! Why do you not just check the parent comment if and set the styles via a computed property? (Something like that: ```:class="{'is-reply': item.parentCommentId}")

Then you could style also the pseudos by the class in the styles directly without hacks.

Reply:
It's very hard to add a pseudo element dynamically. That's the solution I found to handle that. I can remember I tried it with `v-bind:style`, I think also with `:class` but let me check that.

Reply:
Just add the styling with css! Or is there any reason you can’t do that?

Reply:
I can try it again, but I think I also tried it with plain CSS and it didn't work.

Reply:
Obviously for pseudo elements its not that easy to change properties than for normal css elements. I was unable to succeed with another way.

Reply:
My fault, I was able to do it with CSS.